src/components/TeacherComponents/SearchFilters/SearchFilters.tsx

```diff
@@ -36,7 +30,7 @@ const SearchFilters: FC<SearchFiltersProps> = (props) => {
   return (
     <OakThemeProvider theme={oakDefaultTheme}>
       <OakBox>
-        <StyledFieldset>
+        <OakFieldset>
           <OakBox
```

Comment:
I don't often comment on code! Looks great. I am noticing one difference with the behaviour of the screen reader and not sure if it is caused by there only being one option in this fieldset, or because this field set is different than the others. This one is <FIeldset><Box>, the others are <Box><Fieldset>. 

Is it much hassle to switch them round just so we can see if it improves form control readout? Happy to hop on a call and show you what I mean, annoyingly I can't screen capture what is being said!